### ISSUE TITLE
Add coefs to plot

### ISSUE THREAD

**Issue by mattansb** [strengejacke/ggeffects#195]
Suggestion: add the slope coefficients to the plot.
(This has been requested by some clients several times, figured why not...)

**Comment by strengejacke** [comment]
Only for linear models / linear associations?

**Comment by mattansb** [comment]
Yeah - unless you can think of some way to expand this to non-linear cases?

**Comment by mattansb** [comment]
I'm taking about something like this:

![image](https://user-images.githubusercontent.com/35330040/108674644-45fa4480-74ee-11eb-8c21-d15b10b501a4.png)

**Comment by Maxouzzz** [comment]
Hello. 
I am new to the package and under the supervision of [@katossky / @JMeyer31] I am willing to solve this
 issue, as part of an assignment at the ENSAI school of statistical engineering (France). We have read the contibuting guidelines and I will try to be as autonomous as possible but we may ask for guidance from now and then. We have a solid background in statistics and a reasonable one in computer science.
If any maintainer thinks this issue is too ambitious for a beginner, please tell us so and- if you have time- guide us to a project / issue were our help maybe more useful.
Thank you for your consideration.

**Comment by strengejacke** [comment]
I'm happy for any help on open issues, so please feel free to open a pull request! I can assist when necessary and where possible.
